pkg/controller/atlasdeployment/atlasdeployment_controller.go

```diff
@@ -84,7 +83,7 @@ type AtlasDeploymentReconciler struct {
 
 // +kubebuilder:rbac:groups="",namespace=default,resources=events,verbs=create;patch
 
-func (r *AtlasDeploymentReconciler) Reconcile(context context.Context, req ctrl.Request) (ctrl.Result, error) {
+func (r *AtlasDeploymentReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
```

Comment:
Do not agree with this change. It is true that `ctx` is used for `context.Context` in most projects and it was I bit confusing for me too. But this project was created with a different notation and short `ctx` is used for custom object as it is more common.

I think it might be a good idea to refactor this and maybe change the naming a bit. But this has to be a global change for the entire project, otherwise it is confusing. I also think `customContext` is too long and there is actually no reason for our context to even be called "context", and we can just switch it to `state` or something along these lines.

We also don't use `context.Context` properly. So such refactoring with proper `context` usage might be a separate change which we discuss and implement in the future.

Reply:
Done